src/styles/blocks/card.scss

```diff
@@ -0,0 +1,131 @@
+.card {
+  display: flex;
+  box-sizing: border-box;
+  flex-direction: column;
+  flex-shrink: 1;
+  width: 200px;
+  height: auto;
+  font-family: "Roboto", sans-serif;
+  border: 1px solid #f3f3ff;
```

Comment:
It is a good practice to make all colors [variable](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)